cinema/models.py

```diff
@@ -60,6 +60,12 @@ class Meta:
     def __str__(self):
         return self.movie.title + " " + str(self.show_time)
 
+    @property
+    def get_tickets_available(self):
+        return (self.cinema_hall.rows
+                * self.cinema_hall.seats_in_row
+                - len(Ticket.objects.all()))
```

Comment:
The `get_tickets_available` property incorrectly calculates the number of available tickets. It should filter the tickets by the specific `MovieSession` instance rather than using all tickets. Consider using `self.tickets.count()` to get the number of tickets for this session.